frontend/src/features/apportionment/components/residual_seats/ApportionmentResidualSeatsPage.tsx

```diff
@@ -65,11 +61,14 @@ export function ApportionmentResidualSeatsPage() {
   if (seatAssignment) {
     const largestRemainderSteps = seatAssignment.steps.filter(isLargestRemainderAssignmentStep);
     const highestAverageSteps = seatAssignment.steps.filter(isHighestAverageAssignmentStep);
-    const absoluteMajorityChange = seatAssignment.steps
+    const absoluteMajorityReassignment = seatAssignment.steps
       .map((step) => step.change)
       .find((change) => change.changed_by === "AbsoluteMajorityReassignment") as
       | AbsoluteMajorityReassignedSeat
       | undefined;
+    const listExhaustionSteps = seatAssignment.steps.filter(
+      (step) => step.change.changed_by === "ListExhaustionRemoval",
+    );
```

Comment:
```suggestion
    const absoluteMajorityReassignment = seatAssignment.steps.find(isAbsoluteMajorityReassignmentStep);
    const listExhaustionSteps = seatAssignment.steps.filter(isListExhaustionRemovalStep);
```

and then use `absoluteMajorityReassignment.change.pg_assigned_seat` below

Reply:
Fixed in 797d5859b5e6c630a872ea878dd752328b9a20d2